setup/smartos/ansible-playbook.yaml

```diff
@@ -0,0 +1,77 @@
+---
+- hosts: iojs-build-smartos
+
+  remote_user: root
+
+  vars:
+    - ansible_python_interpreter: "/usr/bin/env python"
```

Comment:
Yeah, most pkg-installed stuff lives in `/opt/local/` for SmartOS (whereas other pkgsrc-based distributions go in `/usr/pkg`). I opted for env since that path is added through profile and thought that it'd be a better procedure if anyone else used this playbook as template.